lib/puppetserver/ca/action/sign.rb

```diff
@@ -147,7 +147,7 @@ def sign_requested_certs(certnames,settings)
               success = false
             else
               @logger.err 'Error:'
-              @logger.err "    When download requested for #{result.inspect}"
+              @logger.err "    When attempting to sign certificate request '#{certname}' received:"
```

Comment:
My main reason for this PR was this typo...but while I was here I thought I would standardize all our sign error messaging.

Reply:
Gotcha, I was thinking (hence my comment about revoke) that the updates to puppetserver to support SANs affect all PUTs to the certificate_status endpoint similarly and that this PR re-evaluate all those error messages.

Breaking it up along user facing lines, sign, revoke, create, etc makes sense (vs implementation lines like http methods to an endpoint).